src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs

```diff
@@ -112,9 +111,9 @@ public class ProxyOperationManager
         public ITestRuntimeProvider TestHostManager { get; set; }
 
         /// <summary>
-        /// Gets the proxy operation manager id.
+        /// Gets the proxy operation manager id for proxy test session manager internal organization.
         /// </summary>
-        public Guid Id { get { return this.id; } }
+        public int Id { get; set; } = -1;
```

Comment:
This might cause incompatibilities, as we released this publicly. Please make sure it is backwards compatible with 16.9, 16.10 and 16.11.

Reply:
> There are changes to public APIs, please make sure this can run in LTS releases of VS. Better yet, avoid public API changes if possible.

Thanks for the suggestion, @Haplois. I know the API is public, but the id in case was previously introduced by me as well for the sessions work. This feature (persistent test sessions) is not yet used though, not by VS via TE and for sure not by `dotnet test` (they will have to use the `VsTestConsoleWrapper` in order to take advantage of this feature and they don't).

All in all, as this feature is only expected to be consumed by VS which ships its own `vstest.console` (and those changes are only between `vstest.console` and `TranslationLayer`) there should be no incompatibilities with previous versions.